ci/scripts/run-verilator-tests.sh

```diff
@@ -6,7 +6,9 @@
 set -e
 
 # Increase the test_timeout due to slow performance on CI
-ci/bazelisk.sh test \
+
+./bazelisk.sh query "filter('sim_verilator\_.*\_smoketest', tests(//sw/device/...)) -attr('tags', 'failing_verilator', tests(//sw/device/...)) -attr('tags', 'broken', tests(//sw/device/...))" | \
```

Comment:
Why doesn't the test tag filter remove broken and failing tests?

Reply:
You're totally right -- I didn't think about that. I've removed this from the query